pytorch_lightning/core/datamodule.py

```diff
@@ -247,22 +251,6 @@ def prepare_data(self, *args, **kwargs):
     def setup(self, stage: Optional[str] = None):
         pass
 
-    @abstractmethod
-    def train_dataloader(self, *args, **kwargs) -> DataLoader:
-        pass
-
-    @abstractmethod
-    def val_dataloader(self, *args, **kwargs) -> Union[DataLoader, List[DataLoader]]:
```

Comment:
where all of these goes?

Reply:
already present in inherited `DataHooks`. solves a linked IDE issue.